styles.css

```diff
@@ -7,22 +7,34 @@ body{
   flex-wrap: wrap;
   font-family
 }
-header, footer, aside, .main{
+header, footer, aside, main{
   padding: 1em;
+  display: flex;
```

Comment:
👍 adding `display: flex;` to `.main` fixes the alignment/spacing for `.left` and `.right`